packages/core/utils/lib/sanitize/utils.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const pipeAsync = (...methods) => async data => {
```

Comment:
why is this in sanitize ?

Reply:
Good question, I'll move it to `@strapi/utils`